Makefile

```diff
@@ -102,6 +102,7 @@ SCRIPTS += autoinst.xml
 endif
 
 EFI_EXES := NvmeOfCli.efi VConfig.efi
```

Comment:
I didn't realize that VConfig.efi is a thing.  Now that I am done with my POC I'll have to start more attention.

Reply:
You need it only if you want to test VLAN configurations.

it _might_ be possible to set up VLAN using EFI's HII. I haven't tested that.